tests/repository/metadata/meta_common.py

```diff
@@ -792,12 +797,16 @@ def test_create_image__saved_image(meta: MetadataRepository, created_image):
 
 def test_update_image__ok(meta: MetadataRepository, created_image):
     author = 'hey'
```

Comment:
just to be sure, let's do `author = created_image.author + 'hey'` and same for key

Reply:
Could you please describe what do you want to check here by doing that? I'm sorry but I don't get it. This test ensures that `created_image.author` is not 'hey' before update and is 'hey' after. For me it's a guarantee that update works correctly.

Reply:
it just a paranoia for this case: someone changes `created_image` fixture to have 'hey' as author and this test will test nothing

Reply:
for this we have `assert created_image.params.key != key` before `update_image` call